addon_observatorium.yaml

```diff
@@ -0,0 +1,181 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: api
+    app.kubernetes.io/instance: observatorium-xyz
+    app.kubernetes.io/name: observatorium-api
+    app.kubernetes.io/part-of: observatorium
+    app.kubernetes.io/version: main-2022-03-03-v0.1.2-139-gb3a1918
+  name: observatorium-xyz-observatorium-api
+---
+apiVersion: v1
+data:
+  rbac.yaml: |-
+    "roleBindings":
+      - "name": "test-oidc"
+        "roles":
+          - "read-write"
+        "subjects":
+          - "kind": "user"
+            "name": "user"
+    "roles":
+      - "name": "read-write"
+        "permissions":
+          - "read"
+          - "write"
+        "resources":
+          - "logs"
+          - "metrics"
+          - "traces"
+        "tenants":
+          - "test-oidc"
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/component: api
+    app.kubernetes.io/instance: observatorium-xyz
+    app.kubernetes.io/name: observatorium-api
+    app.kubernetes.io/part-of: observatorium
+    app.kubernetes.io/version: main-2022-03-03-v0.1.2-139-gb3a1918
+  name: observatorium-xyz-observatorium-api
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app.kubernetes.io/component: api
+    app.kubernetes.io/instance: observatorium-xyz
+    app.kubernetes.io/name: observatorium-api
+    app.kubernetes.io/part-of: observatorium
+    app.kubernetes.io/version: main-2022-03-03-v0.1.2-139-gb3a1918
+  name: observatorium-xyz-observatorium-api
+stringData:
+  tenants.yaml: |-
+    "tenants":
+    - "id": "1610b0c3-c509-4592-a256-a1871353dbfa"
+      "name": "test-oidc"
+      "oidc":
+        "clientID": "tempo"
```

Comment:
First I had here `user` which matches the `clientID` configured in Hydra  - `'{"audience":["tempo"],"client_id":"user","client_secret":"secret","grant_types":["client_credentials"],"token_endpoint_auth_method":"client_secret_basic"}'`

However it didn't work and the API gateway was throwing `level=debug name=observatorium ts=2022-10-11T13:04:16.561663802Z caller=oidc.go:357 msg="failed to verify ID token" err="oidc: expected audience \"userko\" got [\"tempo\"]"`

The token looked like:

![Screenshot of JSON Web Tokens - jwt io](https://user-images.githubusercontent.com/5618424/195100122-cb8f9cfb-4911-4db2-b2b8-a141d46103c7.jpg)

Reply:
`clientID` in OBS `tenants.yaml` has to be present in `audience` in hydra config.

* https://github.com/observatorium/observatorium/blob/main/configuration/examples/local/manifests/observatorium/api-secret.yaml#L18
* https://github.com/observatorium/observatorium/blob/main/configuration/examples/local/manifests/hydra/usercreator.yaml